source/_components/vacuum.xiaomi_miio.markdown

```diff
@@ -103,6 +105,74 @@ Enter remote control mode, make one move, stop, and exit remote control mode.
 | `rotation`                |       no | Rotation: between -179 degrees and 179 degrees.       |
 | `duration`                |       no | The number of milliseconds that the robot should move for  |
 
+### {% linkable_title Service `vacuum.xiaomi_clean_zone` %}
+
+Start the cleaning operation in the areas selected for the number of repeats indicated.
+
+| Service data attribute    | Optional | Description                                           |
+|---------------------------|----------|-------------------------------------------------------|
+| `entity_id`               |      yes | Only act on specific robot; default targets all       |
+| `zone`                    |       no | Array of zones. Each zone is an array of 4 integer value. Example: [[23510,25311,25110,26361]] |
+| `repeats`                    |       no | Number of cleaning repeats for each zone between 1 and 3. |
+| `repeats_template`           |       no | Template that rapresents the number of cleaning repeats for each zone between 1 and 3. |
+
+Example of `vacuum.xiaomi_clean_zone` use:
+Inline array:
+```yaml
+automation:
+  - alias: Test vacuum zone3
+    trigger:
+    - event: start
+      platform: homeassistant
+    condition: []
+    action:
+    - service: vacuum.xiaomi_clean_zone
+      data:
+        entity_id: vacuum.xiaomi_vacuum
+        repeats_template: '{{states.input_number.vacuum_passes.state|int}}'
+        zone: [[2555, 2555, 2555, 2555], [2300, 2300, 2300, 2300]]
+```
+Array with inline zone:
+```yaml
+automation:
+  - alias: Test vacuum zone3
+    trigger:
+    - event: start
+      platform: homeassistant
+    condition: []
+    action:
+    - service: vacuum.xiaomi_clean_zone
+      data:
+        entity_id: vacuum.xiaomi_vacuum
+        repeats_template: '{{states.input_number.vacuum_passes.state|int}}'
+        zone: 
+        - [2555, 2555, 2555, 2555]
+        - [2300, 2300, 2300, 2300]
+```
+Array mode:
```

Comment:
Do you think this is good style? Don't provide examples for bad style. I would remove this example.

Reply:
Yes, I think it's a good style.
Lists are representable in many way.